Controller/FileController.php

```diff
@@ -57,7 +57,7 @@ public function __construct(
         ManagerRegistry $registry,
         $managerName,
         $class,
-        $rootPath = '/',
```

Comment:
i am not quite sure how this behaves .. ie. if the default value is ignored atm anyway, as the following parameters do not have default values.